02.calendar/calendar.rb

```diff
@@ -0,0 +1,40 @@
+require 'date'
+require 'optparse'
+
+params = ARGV.getopts("y:m:")
+input_year = "#{params["y"]}"
```

Comment:
```suggestion
input_year = params["y"]
```

文字列の中に入れる必要は無いのでこういう感じでいいと思います〜